clients/web/src/templates/widgets/editCollectionWidget.pug

```diff
@@ -4,10 +4,10 @@
       .modal-header
         button.close(data-dismiss="modal", aria-hidden="true", type="button") &times;
         h4.modal-title
-          if (collection)
-            | Edit collection
-          else
+          if (create)
```

Comment:
I've thought about this more, and I'm concerned that if anyone wants to subclass the `EditCollectionWidget`, the `create` instance variable might not remain strictly equivalent to `collection && collection.id`. Given that `EditCollectionWidget` exposes a public instance variable, `create`, the widget implementation should respect the variable's state as the determinant of whether the widget really is in "create" mode; alternatively, we should not have `create` as an instance variable at all, and always use the (IMO, non-obvious) test `collection && collection.id` to determine the mode of the widget.

I'd call this change a bugfix (to start respecting the `create` variable that always existed) of a part of the code that's not part of the API surface (the injected template variables), so I don't think compatibility is strictly affected. I'd like to keep it as-is.